README.md

```diff
@@ -151,6 +151,15 @@ For example, you can download the latest .tar.gz distribution build of OpenSearc
 
 For plugin artifacts, you can also use the `latest` keyword to get the latest plugin artifacts with a known version. E.g. in order to get performance-analyzer x64 tarball artifacts for 2.1.0, you can obtain it with link `https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.1.0/latest/linux/x64/tar/builds/opensearch/plugins/opensearch-performance-analyzer-2.1.0.0.zip`, which will redirect you to `https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.1.0/5757/linux/x64/tar/builds/opensearch/plugins/opensearch-performance-analyzer-2.1.0.0.zip`.
 
+Here is a table you can check the relationships between available platforms and distributions for artifacts. You can use the combo to replace the corresponding section in the above links.
+
+|  | zip | tar | rpm |
+| -------- | ---------- | ---------- |---------- |
+| linux  |  | :white_check_mark: | :white_check_mark: |
+| windows | :white_check_mark: |  |  |
```

Comment:
I dont think other distributions will ever have tar and rpms right?
Adding these won't be be of much help I believe. Also linux does have zips for plugins so that makes this table confusing.